### DISCUSSION TITLE
是否可以保存overlay类别中元素的数值

### DISCUSSION THREAD

**Discussion by gouzida** [klinecharts/KLineChart#365] [Q&A]
感谢作者对库持续的付出~
我想画完趋势线、价格线之后保存其数值（比如映射到横坐标-时间轴，纵坐标-价格），以便下次打开的时候重新渲染，请问能通过什么方式做到这点呢？ 
更进一步说，是否支持外部数据源控制canvas里面的绘图元素呢？

**Comment by liihuu** [comment]
有get和create方法，看下
